src/software/thunderscope/thunderscope.py

```diff
@@ -3,20 +3,15 @@
 import textwrap
 import shelve
 import signal
-import platform
 import logging
 
 # PyQt5 doesn't play nicely with i3 and Ubuntu 18, PyQt6 is much more stable
```

Comment:
Can we remove/update this comment

Reply:
I did not add the comment, can Arun please verify?

Reply:
you can delete line 8 (the comment is describing the opposite of what the code is actually doing)